RecoEgamma/EgammaElectronProducers/plugins/BuildFile.xml

```diff
@@ -16,9 +16,18 @@
 <use   name="DataFormats/TrackerRecHit2D"/>
 <use   name="DataFormats/TrackingRecHit"/>
 <use   name="DataFormats/ParticleFlowCandidate"/>
+<use   name="RecoParticleFlow/PFClusterTools"/>
 <use   name="hepmc"/>
 <use   name="clhep"/>
 <use   name="root"/>
+
```

Comment:
@Sam-Harper, FYI, these packages are all because of the track extrapolation performed in the new LowPtElectronSeedProducer module.